Makefile

```diff
@@ -311,6 +324,19 @@ check-vera:
 check-cppcheck:
 	$(Q) $(call SHLOG,./tools/check-cppcheck.sh,$(OUT_DIR)/cppcheck.log,Cppcheck)
 
+.PHONY: test-buildoptions
+test-buildoptions:
+	$(Q) for target in $(JERRY_BUILD_OPTIONS_TEST_TARGETS); do \
+           if [[ $$target =~ "mcu" ]]; then \
+             current_options=$$BUILD_OPTIONS_TEST_MCU; \
```

Comment:
We are in the Makefile, so we can use `$(BUILD_OPTIONS_TEST_MCU)`, we don't have to rely on the make variable being exported to a shell variable.